### DISCUSSION TITLE
WNN on different CallPeaks sets

### DISCUSSION THREAD

**Discussion by mzz92** [satijalab/seurat#5126] [Q&A] [answered]
Is there a way to specify which sets of ATAC peaks to use for FindMultiModalNeighbors() and RunUMAP() for WNN analysis?

More specifically: in Signac vignettes/pbmc_multiomic.Rmd example, after CallPeaks, there will be [["ATAC"]] and [["peaks"]]. When FindMultiModalNeighbors() and RunUMAP() are run further down, how do I know whether [["ATAC"]] or [["peaks"]] is being used? 

Is there a way to toggle between the peak sets? Is it just the last set of peaks that's set by DefaultAssay?

Thanks--fantastic resource!

**Comment by timoast** [comment, marked as answer]
`FindMultiModalNeighbors()` uses a set of reduced-dimension spaces, so the set of peaks used will be whatever peaks you have used to construct the LSI dimension reduction. You can store multiple assays with different peak sets in the Seurat object and change between them using the `DefaultAssay()` function, and you can compute multiple LSI dimension reductions by changing the `reduction.name` parameter in `RunSVD()`.

**Reply by mzz92** [reply]
Thanks!
